packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/layout.jsx

```diff
@@ -122,13 +124,15 @@ LayoutBase.propTypes = {
   }),
   visible: PropTypes.bool,
   onHide: PropTypes.func,
+  dateFormat: PropTypes.func,
```

Comment:
dataFormat -> formatData since it is a function

Reply:
@ushkal it's a good idea!